pkg/api/types.go

```diff
@@ -1662,6 +1665,27 @@ type SecretList struct {
 	Items []Secret `json:"items"`
 }
 
+// SecurityContextSpec holds security configuration that will be applied to the pod's containers
+type SecurityContextSpec struct {
+	TypeMeta   `json:",inline"`
+	ObjectMeta `json:"metadata,omitempty"`
+
+	// Capabilities represents the caps to add or remove from all containers in the pod
+	Capabilities Capabilities `json:"capabilities,omitempty"`
+
+	// UserIDMapping is a request to have user ids from X through Y to be mapped to
+	// the container.  For example, map a host user to UID 1000 in the container or
+	// map n host users to 1000-n in the container
+	UserIDMappings []UserIDMappingRange `json:"userIDMappings,omitempty"`
```

Comment:
We shouldn't add userIDMappings yet because user namespaces aren't even implemented in Docker.